test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenHandlerNonParallelRunTests.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Xunit;
```

Comment:
```suggestion
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
﻿using Xunit;
```
We'll have analyzers for stuff like this soon 😊

Reply:
Do you mean is someone already working on adding them or in general?

Reply:
@FuPingFranco You can add this rule (in a separate PR) to the main .editorconfig like we do in MSAL.NET.

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/c9f26e81c114fc9c4bfa4175b7702592e6cb6a90/.editorconfig#L423-L425

Reply:
@pmaytak I meant in general. Since there's other switches that might run into this issue.

Reply:
I'll take Kelly suggestion for this one but it's good to know I can also do it on the  .editorconfig. Thanks!